### PR TITLE
[REF] surveys: remove overview folder

### PR DESCRIPTION
**Pull request by samueljlieber**
Task: [#3189711](https://www.odoo.com/web#id=3189711&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

Targets: 14.0, 15.0, & 16.0

This PR removes the **Overview** folder in Surveys because it is an extra single folder to click through to get to the child pages.

Redirects added for all moved files.

### CODE REVIEW

**Comment by samueljlieber** [comment]
Update in https://github.com/odoo/documentation/commit/5a065b07dce8ddab21346e1cf14f5c50652bd9d4:
- overview.rst --> surveys.rst redirect fixed

**Review by ksc-odoo — APPROVED** [review]

**Comment by samueljlieber** [comment]
Fixed conflict on `redirect.txt` in https://github.com/odoo/documentation/commit/46835ba3b025c8fd37bb3b484eb1f06b2b24d4d6. @StraubCreative ready for your review 🙂

File: content/applications/marketing/surveys.rst [hunk 1]
```diff
@@ -1,6 +1,5 @@
 :nosearch:
 :show-content:
-:hide-page-toc:
 :show-toc:
```
Comment (StraubCreative) [inline content/applications/marketing/surveys.rst:3]: Would delete these too.
Structure it just like events so the top level menu link is a non-indexed page.

**Review by StraubCreative — CHANGES_REQUESTED** [review]
hi @samueljlieber 
Very quick edit which I'll push up in a sec and merge.
Thanks for getting to this so quickly!

**Review by StraubCreative — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**